src/Compilers/CSharp/Portable/Symbols/PublicModel/PropertySymbol.cs

```diff
@@ -58,6 +58,11 @@ IMethodSymbol IPropertySymbol.SetMethod
             get { return _underlying.SetMethod.GetPublicSymbol(); }
         }
 
+        IFieldSymbol IPropertySymbol.BackingField
+        {
+            get { return (_underlying as SourcePropertySymbolBase)?.BackingField.GetPublicSymbol(); }
```

Comment:
I thought the name `BackingField` was more instantly recognizable than `AssociatedField`, so I went with it as soon as I saw precedent.